tests/integration/multinodes.nim

```diff
@@ -146,6 +146,14 @@ template multinodesuite*(name: string, body: untyped) =
         raiseMultiNodeSuiteError "hardhat node not started: " & e.msg
 
       trace "hardhat node started"
+
+      try:
+        await node.postStart()
+      except NodeProcessError as e:
+        raiseMultiNodeSuiteError "cannot run the post start scripts: " & e.msg
```

Comment:
I think it would be better to run `postStart` from `HardhatProcess.start` since it would be not great if someone ran a hardhat process but the MP contract was not deployed. If we need a situation where these scripts are NOT run, we could pass a parameter to `start`.

Reply:
Yeah, I would also go with that route of encapsulating everything in the `start` method.